api/build.gradle

```diff
@@ -239,38 +239,38 @@ dependencies {
   providedCompile group: 'javax.servlet', name: 'javax.servlet-api', version:'3.1.0'
   compile project(":common-api")
   compile 'com.google.api-client:google-api-client-appengine:1.23.0'
-  compile 'mysql:mysql-connector-java:+'
-  compile 'com.google.cloud.sql:mysql-socket-factory:+'
+  compile 'mysql:mysql-connector-java:8.0.11'
+  compile 'com.google.cloud.sql:mysql-socket-factory:1.0.10'
   compile "com.google.appengine:appengine:${GAE_VERSION}"
   compile "com.google.appengine:appengine-api-1.0-sdk:${GAE_VERSION}"
-  compile('org.springframework.boot:spring-boot-starter-web:+') {
+  compile('org.springframework.boot:spring-boot-starter-web') {
```

Comment:
Hm, what does a lack of a version imply here?

Reply:
It gets picked up from the version specified in the plugin.

Reply:
To be extra clear, we can add it here too. Its just a little extraneous.

Reply:
Oh cool, I guess this is what is being used here: https://stackoverflow.com/questions/41676534/how-are-some-gradle-dependencies-working-with-no-version-supplied

Yeah - this is nice, lets definitely leave it off where we can.

Reply:
Good find on the SO link. I didn't know the actual name of what's happening here - good to put a name to something I've only known through practice.